i2c/oled_i2c/img_to_array.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# Converts a greyscale image into a format able to be
+# displayed by the SSD1306 driver in horizontal addressing mode
+
+# usage: ./img_to_array.py <logo.bmp> on a Raspberry Pi
+# or python3 img_to_array.py <logo.bmp>
```

Comment:
"on a Raspberry Pi" is "on Linux, MacOS, etc." :wink: (i.e. any unix-like OS that reads the `#!` line)

Probably easiest to just say:
"# usage: python3 img_to_array.py <logo.bmp>"
as that covers all bases.

Reply:
Ah, yes. I wanted the convenience of just being able to `./` it but I suppose anyone clued up on that will see the shebang and just do it.